src/render/pipeline.rs

```diff
@@ -71,21 +71,24 @@ pub fn extract_tilemaps(
         &TilemapUniformData,
         &Handle<Mesh>,
     )>,
+    images: Res<Assets<Image>>,
 ) {
     let mut extracted_tilemaps = Vec::new();
     for (entity, transform, chunk, tilemap_uniform, mesh_handle) in query.iter() {
-        let transform = transform.compute_matrix();
-        extracted_tilemaps.push((
-            entity,
-            (
-                LayerId(chunk.settings.layer_id),
-                chunk.material.clone(),
-                chunk.settings.mesh_type.clone(),
-                mesh_handle.clone_weak(),
-                tilemap_uniform.clone(),
-                MeshUniform { transform },
-            ),
-        ));
+        if images.get(&chunk.material).is_some() {
+            let transform = transform.compute_matrix();
```

Comment:
if I'm correct than this replaces panic with silent failure when image is not available, is that correct? I think it would be helpful to at least log some sort of warning in these cases.

Reply:
It's quite common in bevy to silently fail in this case. If the asset fails to load the asset system will warn.